packages/ui/package.json

```diff
@@ -46,6 +48,18 @@
         "types": "./dist/components/ui/*.d.ts",
         "default": "./dist/components/ui/*.js"
       },
+      "./theme": {
+        "types": "./dist/src/PenumbraUIProvider/theme.d.ts",
+        "default": "./dist/src/PenumbraUIProvider/theme.js"
+      },
+      "./lib/toast/*": {
```

Comment:
issue: this is the old toast library, it should not be exported. The v2 one is already working in the dex: https://github.com/penumbra-zone/dex-explorer/pull/118/files#diff-e1b5cf1eb29b361c28415989517c8312f1f349ab3e0cac44425ffc02290259c2R62